slurk/models/user.py

```diff
@@ -50,24 +50,29 @@ def join_room(self, room):
         if self.session_id is not None:
             join_room(str(room.id), self.session_id, "/")
 
+            user = dict(id=self.id, name=self.name)
+            room_id = room.id
+
+            def joined():
+                socketio.emit(
+                    "status",
+                    dict(
+                        type="join",
+                        user=dict(id=user["id"], name=user["name"]),
```

Comment:
I noticed after suggesting this, that we could now also simply write `user=user` here. As the dict we created before now looks exactly like the one we want to pass.